### PR TITLE
SideNav Hamburger Animation #519

### PR DESCRIPTION
**Pull request by Chaitanya31612**
Added given hamburger navbar toggler into app.

### CODE REVIEW

**Comment by smaranjitghose** [comment]
hamb
Icon is not displaying

**Review by smaranjitghose — APPROVED** [review]

**Comment by Chaitanya31612** [comment]
> hamb
> Icon is not displaying

image
Strange, it's coming for me and the animation is also working as expected.

**Comment by smaranjitghose** [comment]
side_nav

**Review by anushbhatia — CHANGES_REQUESTED** [review]
image
Align to the right side of the header a

**Review by anushbhatia — CHANGES_REQUESTED** [review]
Put a comment like

``` terminal
/* 
---------------------------------
         TEXT ANIMATIONS
----------------------------------
*/
```
so the file is more readable 
Add comments like these

File: assets/js/app.js [hunk 1]
```diff
@@ -78,5 +80,12 @@ $(function () {
 
 });
```
Comment (smaranjitghose) [inline assets/js/app.js:82]: PLEASE PUT COMMENTS FOR FUNCTION DESCRIPTIONS.

**Review by smaranjitghose — APPROVED** [review]

**Comment by smaranjitghose** [comment]
@Chaitanya31612 I am merging this PR.

If it's okay, I would ask you to give a PR and comment your code (document it) to make it more readable and explainable. This is also one of the goals of this project

**Comment by Chaitanya31612** [comment]
> @Chaitanya31612 I am merging this PR.
> 
> If it's okay, I would ask you to give a PR and comment your code (document it) to make it more readable and explainable. This is also one of the goals of this project

Sure, @smaranjitghose I'll make a pull request with comments for code I've written.

**Pull request merged**